pkgs/desktops/plasma-5/plasma-vault/default.nix

```
@@ -36,6 +37,8 @@ mkDerivation {
     ''-DNIXPKGS_CRYFS=\"${lib.getBin cryfs}/bin/cryfs\"''
 
     ''-DNIXPKGS_FUSERMOUNT=\"${lib.getBin fuse}/bin/fusermount\"''
+
```

```suggestion
    ''-DNIXPKGS_CRYFS=\"${lib.getBin cryfs}/bin/cryfs\"''
    ''-DNIXPKGS_FUSERMOUNT=\"${lib.getBin fuse}/bin/fusermount\"''
```
also drop the new line in line 36